util/package.go

```diff
@@ -26,22 +26,23 @@ var CategoryTitles = map[string]string{
 }
 
 type Package struct {
+	FormatVersion string `config:"format_version" json:"format_version" yaml:"format_version"`
+
 	Name          string  `config:"name" json:"name"`
-	Title         *string `config:"title,omitempty" json:"title,omitempty"`
+	Title         *string `config:"title,omitempty" json:"title,omitempty" yaml:"title,omitempty"`
```

Comment:
Your change made me realise we don't have a license field in here but it is something we have in the manifest. Will open a PR.